admin/views/class-yoast-feature-toggles.php

```diff
@@ -0,0 +1,179 @@
+				'label'           => sprintf( __( 'Enable the XML sitemaps that %s generates.', 'wordpress-seo' ), 'Yoast SEO' ),
+				'read_more_label' => __( 'Read why XML Sitemaps are important for your site.', 'wordpress-seo' ),
+				'read_more_url'   => 'https://yoa.st/2a-',
+				'extra'           => $xml_sitemap_extra,
+				'order'           => 60,
+			),
+			(object) array(
+				/* translators: %s: Ryte */
+				'name'            => sprintf( __( '%s integration', 'wordpress-seo' ), 'Ryte' ),
+				'setting'         => 'onpage_indexability',
+				/* translators: 1: Ryte */
+				'label'           => sprintf( __( '%1$s will check weekly if your site is still indexable by search engines and Yoast SEO will notify you when this is not the case.', 'wordpress-seo' ), 'Ryte' ),
+				/* translators: %s: Ryte */
+				'read_more_label' => sprintf( __( 'Read more about how %s works.', 'wordpress-seo' ), 'Ryte ' ),
+				'read_more_url'   => 'https://yoa.st/2an',
+				'order'           => 70,
+			),
+			(object) array(
+				'name'    => __( 'Admin bar menu', 'wordpress-seo' ),
+				'setting' => 'enable_admin_bar_menu',
+				/* translators: 1: Yoast SEO */
+				'label'   => sprintf( __( 'The %1$s admin bar menu contains useful links to third-party tools for analyzing pages and makes it easy to see if you have new notifications.', 'wordpress-seo' ), 'Yoast SEO' ),
+				'order'   => 80,
+			),
+			(object) array(
+				'name'    => __( 'Security: no advanced settings for authors', 'wordpress-seo' ),
+				'setting' => 'disableadvanced_meta',
+				'label'   => sprintf(
+					/* translators: 1: Yoast SEO, 2: translated version of "Off" */
+					__( 'The advanced section of the %1$s meta box allows a user to remove posts from the search results or change the canonical. These are things you might not want any author to do. That\'s why, by default, only editors and administrators can do this. Setting to "%2$s" allows all users to change these settings.', 'wordpress-seo' ),
+					'Yoast SEO',
+					__( 'Off', 'wordpress-seo' )
+				),
+				'order'   => 90,
+			),
+		);
+
+		/**
+		 * Filter to add feature toggles from add-ons.
+		 *
+		 * @param array $feature_toggles Array with feature toggle objects where each object should have a `name`, `setting` and `label` property.
+		 */
+		$feature_toggles = apply_filters( 'wpseo_feature_toggles', $feature_toggles );
+
+		$feature_toggles = array_map( array( $this, 'ensure_toggle' ), $feature_toggles );
+		usort( $feature_toggles, array( $this, 'sort_toggles_callback' ) );
+
+		return $feature_toggles;
+	}
+
+	/**
+	 * Ensures that the passed value is a Yoast_Feature_Toggle.
+	 *
+	 * @param Yoast_Feature_Toggle|object|array $toggle_data Feature toggle instance, or raw object or array
+	 *                                                       containing feature toggle data.
+	 * @return Yoast_Feature_Toggle Feature toggle instance based on $toggle_data.
+	 */
+	protected function ensure_toggle( $toggle_data ) {
```

Comment:
I am not a big fan of having an attribute that can have different types.

Reply:
Agreed, in this specific case though, the only reason the method exists is to ensure that feature toggles that are the result of people using the filter in a wrong way are still recognized properly.